.travis.yml

```diff
@@ -26,3 +27,7 @@ matrix:
     - rvm: 2.1
       gemfile: gemfiles/rails_edge.gemfile
       env: DEFER_GC=false RAILS_EDGE=true
+    - rvm: 2.2.0
+      gemfile: gemfiles/rails_3.2.gemfile
```

Comment:
Then we should have this as an allowed failure.

For the page_title_helper, we did it like this:
https://github.com/lwe/page_title_helper/blob/master/.travis.yml